packages/manager/apps/veeam-backup/src/pages/dashboard/Dashboard.page.tsx

```diff
@@ -54,6 +57,14 @@ export default function DashboardPage() {
   const { t } = useTranslation('dashboard');
   const displayName = getVeeamBackupDisplayName(data?.data);
 
+  const consumedVms = useMemo(() => {
+    if (!Array.isArray(consumptions)) return null;
```

Comment:
```suggestion
    if (!Array.isArray(consumptions)) return '-';
```

May be better to show `- VMs` inst end `VMs` during loading of consumed data